plugins/modules/ec2_vpc_route_table.py

```diff
@@ -356,23 +351,23 @@ def find_subnets(connection, module, vpc_id, identified_subnets):
     if subnet_ids:
         filters = ansible_dict_to_boto3_filter_list({"vpc-id": vpc_id})
         try:
-            subnets_by_id = describe_subnets_with_backoff(connection, SubnetIds=subnet_ids, Filters=filters)
+            subnets_by_id = describe_subnets(connection, SubnetIds=subnet_ids, Filters=filters)
         except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError) as e:
```

Comment:
Should this and the calls below be updated to check for `AnsibleEC2Error`?

Reply:
Updated